### PR TITLE
feat(gamble): [OWL-27896248] append current balance to on success messages

### PR DESCRIPTION
**Pull request by alvinb**
- appends "Your current balance `{amount}`" when users successfully gives money or when they win using the gamble command

### CODE REVIEW

File: src/discord/commands/gamble.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const Gamble = {
       if (result === 'win') {
         points += user.cash;
         await interaction.reply({
-          content: `You won ${user.cash} ${CONFIG.CURRENCY.PLURAL}! :moneybag:`,
+          content: `You won ${user.cash} ${CONFIG.CURRENCY.PLURAL}! :moneybag: Your cash balance: ${points} :coin:.`,
```
Comment (devthena) [inline src/discord/commands/gamble.ts:85]: Let's remove the period at the end of the sentence for all of them

File: src/discord/commands/give.ts [hunk 2]
```diff
@@ -74,17 +74,19 @@ export const Give = {
       ?.collection(Bots.env.MONGODB_USERS)
       .updateOne(
         { discord_id: recipient.discord_id },
-        { $set: { cash: (recipient.cash += amount) } }
+        { $set: { points: (recipient.cash += amount) } }
```
Comment (devthena) [inline src/discord/commands/give.ts:77]: We should keep the `cash` key the same. We removed `points` from the new collection

File: src/discord/commands/give.ts [hunk 3]
```diff
@@ -74,17 +74,19 @@ export const Give = {
...
       );
 
     await Bots.db
       ?.collection(Bots.env.MONGODB_USERS)
       .updateOne(
         { discord_id: user.discord_id },
-        { $set: { cash: (user.cash -= amount) } }
+        { $set: { points: (user.cash -= amount) } }
```
Comment (devthena) [inline src/discord/commands/give.ts:84]: Same as above

**Review by devthena — CHANGES_REQUESTED** [review]

File: src/discord/commands/give.ts (hunk 2 above)
Comment (alvinb) [reply src/discord/commands/give.ts:77]: ohh whatta. i dont remember making this change. mustve been a merge error.
good catch

File: src/discord/commands/give.ts [hunk 4]
```diff
@@ -84,7 +84,9 @@ export const Give = {
         { $set: { cash: (user.cash -= amount) } }
       );
 
-    await interaction.reply({ content: replies.success });
+    await interaction.reply({
+      content: `${replies.success}. Your cash balance: ${user.cash} :coin:.`,
```
Comment (devthena) [inline src/discord/commands/give.ts:88]: Let's also remove the period at the end for this

File: src/twitch/commands/gamble.ts [hunk 5]
```diff
@@ -51,7 +51,7 @@ export const onGamble = async (
       points += user.cash;
       Bots.twitch.say(
         channel,
-        `${user.twitch_username} won ${user.cash} ${CONFIG.CURRENCY.PLURAL}! :moneybag:`
+        `${user.twitch_username} won ${user.cash} ${CONFIG.CURRENCY.PLURAL}! :moneybag: Your cash balance: ${points}.`
```
Comment (devthena) [inline src/twitch/commands/gamble.ts:54]: Same as above, let's remove all periods at the end of the `:coin:` emoji

**Review by devthena — CHANGES_REQUESTED** [review]

File: src/twitch/commands/gamble.ts (hunk 5 above)
Comment (alvinb) [reply src/twitch/commands/gamble.ts]: whoops forgot about this one. 

addressed it here: https://github.com/devthena/little-owl/pull/26/commits/3d2d891981174f9880b41a05ccc944fdd1b44956

File: src/discord/commands/give.ts (hunk 4 above)
Comment (alvinb) [reply src/discord/commands/give.ts]: whoops. addressed here:
https://github.com/devthena/little-owl/pull/26/commits/744a4c8f868019bc122da3c73a48740a73e170ce

**Review by devthena — APPROVED** [review]

**Pull request merged**